public/js/apps/management/app.jsx

```diff
@@ -118,7 +120,18 @@ export default class ManagementApp extends Component {
           {...boundPayMethodActions}
         />
       ));
-    } else if (connector.management.view === 'SHOW_SUBSCRIPTIONS') {
+    } else if (connector.management.view === 'SHOW_CONFIRM_DEL_PAY_METHOD') {
+      children.push((
+        <ConfirmDelPayMethod
+          payMethods={connector.user.payMethods}
+          payMethodUri={connector.user.tmp.payMethodUri}
```

Comment:
that way we could wipe it out when switching tabs which is probably better than wiping it out when switching users